.circleci/config.yml

```diff
@@ -54,20 +54,22 @@ commands:
           command: cmd/consensus
 
 jobs:
-  linux-gcc-7:
+  linux-gcc-9:
     environment:
       BUILD_TYPE: Debug
       CLANG_COVERAGE: OFF
     machine:
       image: ubuntu-2004:202010-01
     steps:
       - run:
-          name: "Switch to GCC 7"
+          name: "Switch to GCC 9"
```

Comment:
I believe Ubuntu 20.04 comes with gcc 9 by default, so we can probably skip this step entirely.

Reply:
Ouch ... thought it was a 18.04 LTS. My bad